### ISSUE TITLE
Figure out why `parsedatetime==2.5` breaks collate tests

### ISSUE THREAD

**Issue by shaycrk** [dssg/triage#721]
Currently pegged `parsedatetime` (a csvkit/agate dependency) to version 2.4 to avoid the error, but ideally could remove this specific dependency if we tracked down the underlying issue. For a stacktrace with the bug, see [this TravisCI run](https://travis-ci.com/dssg/triage/jobs/266390826) (pasted below, too) while collecting test cases.

```
collected 385 items / 1 errors                                                 
==================================== ERRORS ====================================
_________ ERROR collecting src/tests/collate_tests/test_integration.py _________
src/tests/collate_tests/test_integration.py:20: in <module>
    cache_initialized_db=True, on_initialized=initialize_db.handler
.tox/py3/lib/python3.6/site-packages/testing/common/database.py:41: in __init__
    init_handler(self.cache)
src/tests/collate_tests/initialize_db.py:14: in handler
    load_data(connection)
src/tests/collate_tests/initialize_db.py:37: in load_data
    check=True,
/opt/python/3.6.7/lib/python3.6/subprocess.py:418: in run
    output=stdout, stderr=stderr)
E   subprocess.CalledProcessError: Command '['csvsql', '-v', '--no-constraints', '--tables', 'food_inspections', '--insert', '--db', 'postgresql://postgres@127.0.0.1:58119/test', '/home/travis/build/dssg/triage/src/tests/collate_tests/food_inspections_subset.csv']' returned non-zero exit status 1.
------------------------------- Captured stderr --------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dssg/triage/.tox/py3/bin/csvsql", line 8, in <module>
    sys.exit(launch_new_instance())
  File "/home/travis/build/dssg/triage/.tox/py3/lib/python3.6/site-packages/csvkit/utilities/csvsql.py", line 221, in launch_new_instance
    utility.run()
  File "/home/travis/build/dssg/triage/.tox/py3/lib/python3.6/site-packages/csvkit/cli.py", line 118, in run
    self.main()
  File "/home/travis/build/dssg/triage/.tox/py3/lib/python3.6/site-packages/csvkit/utilities/csvsql.py", line 115, in main
    self._failsafe_main()
  File "/home/travis/build/dssg/triage/.tox/py3/lib/python3.6/site-packages/csvkit/utilities/csvsql.py", line 150, in _failsafe_main
    **self.reader_kwargs
  File "/home/travis/build/dssg/triage/.tox/py3/lib/python3.6/site-packages/agate/table/from_csv.py", line 88, in from_csv
    return Table(rows, column_names, column_types, row_names=row_names)
  File "/home/travis/build/dssg/triage/.tox/py3/lib/python3.6/site-packages/agate/table/__init__.py", line 109, in __init__
    self._column_types = column_types.run(rows, self._column_names)
  File "/home/travis/build/dssg/triage/.tox/py3/lib/python3.6/site-packages/agate/type_tester.py", line 110, in run
    if len(row) > i and not column_type.test(row[i]):
  File "/home/travis/build/dssg/triage/.tox/py3/lib/python3.6/site-packages/agate/data_types/base.py", line 29, in test
    self.cast(d)
  File "/home/travis/build/dssg/triage/.tox/py3/lib/python3.6/site-packages/agate/data_types/date.py", line 76, in cast
    (value, ctx, _, _, matched_text), = self.parser.nlp(d, sourceTime=ZERO_DT)
  File "/home/travis/build/dssg/triage/.tox/py3/lib/python3.6/site-packages/parsedatetime/__init__.py", line 2233, in nlp
    version)
  File "/home/travis/build/dssg/triage/.tox/py3/lib/python3.6/site-packages/parsedatetime/__init__.py", line 1844, in parse
    retS, retTime, matched = parseMeth(s, sourceTime)
  File "/home/travis/build/dssg/triage/.tox/py3/lib/python3.6/site-packages/parsedatetime/__init__.py", line 1408, in _partialParseQUnits
    sourceTime = self._evalQUnits(parseStr, sourceTime)
  File "/home/travis/build/dssg/triage/.tox/py3/lib/python3.6/site-packages/parsedatetime/__init__.py", line 1100, in _evalQUnits
    sourceTime = self._buildTime(sourceTime, quantity, modifier, units)
  File "/home/travis/build/dssg/triage/.tox/py3/lib/python3.6/site-packages/parsedatetime/__init__.py", line 389, in _buildTime
    ctx.updateAccuracy(realunit)
  File "/home/travis/build/dssg/triage/.tox/py3/lib/python3.6/site-packages/parsedatetime/context.py", line 137, in updateAccuracy
    acc = self._ACCURACY_REVERSE_MAPPING[acc]
KeyError: 's'
```

**Comment by thcrock** [comment]
The comments here look relevant https://github.com/bear/parsedatetime/pull/233

**Comment by thcrock** [comment]
@nanounanue @shaycrk They released parsedatetime 2.6 which should fix this issue.

The latest parsedatetime actually got bundled in #753 so this issue should be closed if that PR gets merged.

**Comment by nanounanue** [comment]
@shaycrk I am checking this in several related branches. I will take care, don't worry

**Comment by nanounanue** [comment]
Closed in a8a8388d9a04a1e39bb918935d0b9262bb85a703
